### PR TITLE
fix telemetry doc

### PR DESCRIPTION
**Pull request by zirain**
This PR try to improve [Telemetry API](https://preliminary.istio.io/latest/docs/reference/config/telemetry/#AccessLogging-Filter) doc, I'm not sure this is the right way, feel free to close this if not.

image

image

image

### CODE REVIEW

**Comment by ericvn** [comment]
I did build with the changes as they should be copied to istio.io to verify the changes:

Screen Shot 2021-12-28 at 10 26 16 AM
Screen Shot 2021-12-28 at 10 26 05 AM
Screen Shot 2021-12-28 at 10 25 39 AM

**Comment by ericvn** [comment]
@zirain Would you add a release note saying that formatting was fixed for some entries on the particular page. That will fix the failing test. Thanks.

File: telemetry/v1alpha1/telemetry.proto [hunk 1]
```diff
@@ -36,9 +36,10 @@ import "google/protobuf/wrappers.proto";
 // any given workload.
 //
 // The hierarchy of Telemetry configuration is as follows:
-// 1. Workload-specific configuration
-// 1. Namespace-specific configuration
-// 1. Root namespace configuration
+//
+// - Workload-specific configuration
+// - Namespace-specific configuration
+// - Root namespace configuration
```
Comment (howardjohn) [inline telemetry/v1alpha1/telemetry.proto:42]:
```suggestion
// 1. Workload-specific configuration
// 2. Namespace-specific configuration
// 3. Root namespace configuration
```

So it is still numbered?

**Review by howardjohn — APPROVED** [review]

**Comment by ericvn** [comment]
It seems the suggested change will just change from a `ul` to a `ol` in the html (didn't check the others) which will show correctly numbered:
Screen Shot 2021-12-28 at 12 47 20 PM

**Comment by zirain** [comment]
@ericvn please take a look again

**Comment by zirain** [comment]
BTW, @ericvn how to do istio.io build locally? where is something like guideline?

```
I did build with the changes as they should be copied to istio.io to verify the changes:
```

**Comment by ericvn** [comment]
@zirain I don't know that we call out specifically how to build the docs with istio/api changes anywhere in a readme.

There is a nightly job which copies some files from other repos into the istio.io repo. Some of those files are the `pb.html` which have a location specified in the header, in this case the new file has: `https://istio.io/docs/reference/config/telemetry.html`. I know that this gets copied over a file in the istio.io repo, namely content/en/docs/reference/config/telemetry/index.html. There are some other substitutions made to things like URLs, so it's not a 100% direct replacement. I just use a compare tool to pull the changes noted in the istio/api repo or you could tweak the scripts/grab_reference_docs.sh in the repo to pull from your local repo as well. Once the file(s) are updated in the istio.io repo, you can run `make serve` which will create a local server which you can browse to see any changes.

File: releasenotes/notes/fix-telemetry-doc.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+
+kind: feature
+area: telemetry
```
Comment (ericvn) [inline releasenotes/notes/fix-telemetry-doc.yaml:4]:
```suggestion
area: documentation
```

File: releasenotes/notes/fix-telemetry-doc.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+issue: []
+
+releaseNotes:
+- |
+    **Fixed** formatting was fixed for some entries on the particular page.
```
Comment (ericvn) [inline releasenotes/notes/fix-telemetry-doc.yaml:9]:
```suggestion
    **Fixed** formatting of telemetry configuration reference page.
```

File: releasenotes/notes/fix-telemetry-doc.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+
+kind: feature
```
Comment (ericvn) [inline releasenotes/notes/fix-telemetry-doc.yaml:3]:
```suggestion
kind: bug-fix
```

**Review by ericvn — COMMENTED** [review]
Some changes in the release notes. You could just remove the issue: line as well.

**Review by ericvn — APPROVED** [review]

**Pull request merged**